content/token-swap.md

```diff
@@ -0,0 +1,539 @@
+    const scroogeATA = await token.getAssociatedTokenAddress(ScroogeCoinMint, publicKey)
+    const tokenAccountPool = await token.getAssociatedTokenAddress(pool_mint, publicKey)
+
+    const transaction = new Web3.Transaction()
+
+    let account = await connection.getAccountInfo(tokenAccountPool)
+
+    if (account == null) {
+        const createATAInstruction =
+            token.createAssociatedTokenAccountInstruction(
+                publicKey,
+                tokenAccountPool,
+                publicKey,
+                pool_mint
+            )
+        transaction.add(createATAInstruction)
+    }
+}
+```
+
+Finally, we can create and add the deposit instruction using the `spl-token-swap` libary's `TokenSwap.depositAllTokenTypesInstruction` helper function. With that instruction added to the transaction, we can submit the transaction.
+
+```tsx
+const handleTransactionSubmit = async () => {
+    if (!publicKey) {
+        alert('Please connect your wallet!')
+        return
+    }
+
+    const kryptATA = await token.getAssociatedTokenAddress(kryptMint, publicKey)
+    const scroogeATA = await token.getAssociatedTokenAddress(ScroogeCoinMint, publicKey)
+    const tokenAccountPool = await token.getAssociatedTokenAddress(poolMint, publicKey)
+
+    const transaction = new Web3.Transaction()
+
+    let account = await connection.getAccountInfo(tokenAccountPool)
+
+    if (account == null) {
+        const createATAInstruction =
+            token.createAssociatedTokenAccountInstruction(
+                publicKey,
+                tokenAccountPool,
+                publicKey,
+                poolMint
+            )
+        transaction.add(createATAInstruction)
+    }
+
+    const instruction = TokenSwap.depositAllTokenTypesInstruction(
+        tokenSwapStateAccount,
+        swapAuthority,
+        publicKey,
+        kryptATA,
+        scroogeATA,
+        poolKryptAccount,
+        poolScroogeAccount,
+        poolMint,
+        tokenAccountPool,
+        TOKEN_SWAP_PROGRAM_ID,
+        token.TOKEN_PROGRAM_ID,
+        poolTokenAmount,
+        100e9,
+        100e9
+    )
+
+    transaction.add(instruction)
+    
+    try {
+        let txid = await sendTransaction(transaction, connection)
+        alert(`Transaction submitted: https://explorer.solana.com/tx/${txid}?cluster=devnet`)
+        console.log(`Transaction submitted: https://explorer.solana.com/tx/${txid}?cluster=devnet`)
+    } catch (e) {
+        console.log(JSON.stringify(e))
+        alert(JSON.stringify(e))
+    }
+}
+```
+
+With the exception of the user’s `publickey` and their derived associated token accounts (for Krypt Coin, Scrooge Coin, and the pool's LP token), notice that all the accounts are constants for this swap pool and are defined in the `const.ts` file.
+
+At this point, you should be able to airdrop yourself some tokens and then deposit them into the swap pool!
+
+### 3. Create the Withdrawal Instruction
```

Comment:
@ZYJLiu I got up to this point with editing the demo. Go through the rest of the demo and get the project working using the same conventions I used in `Deposit.tsx`, and update the demo instructions here to match.